antenna-documentation/src/site/markdown/sw360-data-model.md.vm

```diff
@@ -64,8 +64,10 @@ In the following section there is an outline what data of SW360 ${docNameCap} ma
   | Release Tag Url | This is a link to the OSS repos tag of the release used stored as external id with key `orig_repo`. |
   | Software Heritage ID | A release ID in software heritage in the format "swh:1:rel:*", stored as external id with key `swh`. |
   | Copyrights | Copyrights of the release, given in a String. Individual copyrights are separated by line. Stored in the additional data section with key `copyrights` |
-  | Change Status | An enumeration with values `changed` or `as-is` stored as additional data with key `change_status`. A changed component release should be stored as own release with an appendix to the original version, e.g. `1.2.3_modified_xyz`. Currently the implementation can only handle `as-is` components. |
+  | Change Status | An enumeration with values `CHANGED` or `AS_IS` stored as additional data with key `change_status`. A changed component release should be stored as own release with an appendix to the original version, e.g. `1.2.3_modified_xyz`. Currently the implementation can only handle `AS_IS` components. |
```

Comment:
Commit message: Please make the reference to the issue go last (between body and footer) and use one of the [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).